src/spec_assign.cpp

```diff
@@ -36,12 +36,12 @@ struct train_data trainers[] = {
 /* if they teach < 5 skills, put a 0 in the remaining skill slots    */
 /* available levels are NEWBIE (4), AMATEUR (6/7), and ADVANCED (10/11)   */
 struct teach_data metamagict[] = {
-                       { 24806, { META_REFLECTING, 0, 0, 0, 0, 0, 0, 0 }, "", 0 },
-                         { 4250, { META_CLEANSING, 0, 0, 0, 0, 0, 0, 0 }, "", 0 },
-                         { 4251, { META_MASKING, 0, 0, 0, 0, 0, 0, 0 }, "", 0 },
+                       { 24806, { META_MASKING, 0, 0, 0, 0, 0, 0, 0 }, "", 0 },
```

Comment:
What's the rationale behind these changes?

Reply:
Masking is the most common and assigned as a generalist to the more obviously positioned trainers, the two reflecting trainers are identical mobs, cleansing was given to a mob who occupies one of the solitary clean rooms in the middle of grotesque squalor.

Essentially, flavor.